### DISCUSSION TITLE
Adjust opus bit rate and ptime

### DISCUSSION THREAD

**Discussion by mmoreton** [InnovateAsterisk/Browser-Phone#244] [Q&A]
Hi,

Is it possible to customize the parameters of opus codec? I would like to use opus with a bitrate of 8kbps instead of 48kbps and modify the default value of its ptime, but I don't know if it's possible and where I should configure it. 

Thanks

**Comment by InnovateAsterisk** [comment]
I found that it uses 16kb/s normally. It’s probably to do with what your server can accept. 

When you are in a call, lift the call controls panel (bottom of the call window), and click the graph icon to get call stats. Here you can see getusermedia stats like packets and bytes sent etc.
